fe/fe-core/src/main/java/com/starrocks/catalog/TableProperty.java

```diff
@@ -133,6 +135,9 @@ public static String valueList() {
     // Indicates which tables do not listen to auto refresh events when load
     private List<TableName> excludedTriggerTables;
 
+    // This property only applies to materialized views
+    private List<String> mvSortKeys;
```

Comment:
Why doesn't the OLAP table require a sort key property, and how does it manage this absence?

Reply:
The sort keys of table and MV are different:
- Sort key of table: implicit for duplicate key, explicit for primary key, so there's no unified syntax and metadata
- Sort key of MV: could be explicited  specified by `ORDER BY`, or chosen automatically 

Why recording the `mvSortKeys` metadata here? Because there's no syntax or metadata to specify the sort key of a  duplicated key table.

When would it record the `mvSortKeys`? Only if explicitly specify the `ORDER BY`